linode/manager
# M3-5609 PasswordInput component story

## Description

- Convert PasswordInput story to MDX
- Remove unused styles in PasswordInput component

## How to test

Run Storybook and visit http://localhost:6006/?path=/docs/ui-elements-password-input--password-input. Run the app and ensure there are no regressions with the PasswordInput component.
